apps/docs-app/docs/integrations/ionic/index.md

```diff
@@ -210,6 +208,91 @@ export default defineConfig(({ mode }) => {
 @import '@ionic/angular/css/palettes/dark.system.css';
 ```
 
+### Server Side Rendering Caveat
+
+Ionic Framework [doesn't support the Angular's new Client Hydration](https://github.com/ionic-team/ionic-framework/issues/28625#issuecomment-1843919548), as Angular [doesn't support SSR with web components](https://github.com/angular/angular/issues/52275), and when they are supported, work has to be done on the Stencil components to enable it. So right now there are three options to handle this:
+
+1. Remove `provideClientHydration()` from `app.config.ts` providers.
+
+   - This removes the new client hydration mechanism from Angular and reverts to the previous one, which will cause a flicker when re-rendering the DOM from the client.
+
+   ```ts
+   import { RouteReuseStrategy, provideRouter } from '@angular/router';
+   import {
+     IonicRouteStrategy,
+     provideIonicAngular,
+   } from '@ionic/angular/standalone';
+
+   export const appConfig: ApplicationConfig = {
+     providers: [
+       provideFileRouter(),
+       //provideClientHydration(), // remove this.
+       provideHttpClient(withFetch()),
+       { provide: RouteReuseStrategy, useClass: IonicRouteStrategy },
+       provideIonicAngular(),
+     ],
+   };
+   ```
+
+2. Add `ngSkipHydration` attribute to the `ion-app` tag.
+
+   - This will disable the client hydration mechanism for the `ion-app` element and children, but will continue to use client hydration on other elements. This will also cause a flicker in the page for the Ionic components. This is not that helpful for other elements/components as, with Ionic apps, all your Ionic components exist inside the `ion-app` tag.
+
+     ```ts
+     import { Component } from '@angular/core';
+     import { IonApp, IonRouterOutlet } from '@ionic/angular/standalone';
+
+     @Component({
+       selector: 'demo-root',
+       standalone: true,
+       imports: [IonApp, IonRouterOutlet],
+       template: `
+         <ion-app ngSkipHydration>
+           <ion-router-outlet></ion-router-outlet>
+         </ion-app>
+       `,
+     })
+     export class AppComponent {}
+     ```
+
+3. Disable SSR completely
+
+   - Disable SSR in the `vite.config.ts` file. This will disable SSR in your whole app but **will eliminate the flickering**.
```

Comment:
```suggestion
   - Disable SSR in the `vite.config.ts` file. This will disable SSR in your whole app and **will eliminate the flickering**.
```
minor: "but" makes it sound as if eliminating the flicking is bad.

Reply:
@ajitzero Rephrased it to give emphasis on what is the disadvantages of disabling it.